Cookbook/Technical-Documents/WhatIDontKnow.md

```diff
@@ -0,0 +1,183 @@
+# What I don't know, but should
+
+## Congratulations you’ve joined Babylon...
+Welcome to the amazing iOS team! 
+Babylon is a fast-growing company as you could see we are onboarding ~ 30 people weekly many of them engineers like yourself. Our team is scattered across three offices: London West, London East and US office. On top of that, we have fully remote teammates. As you can imagine this poses almost biblical challenges, but don’t worry :)
+
+## Like others before you...
+The first week may be difficult, all of the reading and chatting make for a bit of a mind-numbing first experience. We don’t promise the information firehouse stops after the first day, but just hear what people who get through are saying:
+
+Q: What did you like during your first week?
+> “1-on-1 with Rui and squad PMs, meet team, talk a lot within the squad, pair with a couple of people on iOS team. Gone out to drinks with the squad :)”
+
+> “I found the team to be very helpful in the first while becoming familiar with the code ...”
+
+> “Working in my squad on real tickets from day one”
+
+> “Quickly started working on an actual project, smooth onboarding because of the iOS playbook, people being extremely helpful and welcoming, 1st day with the team”
+
+> “Team welcoming both iOS and squad ”
+
+## You’ll never walk alone…
+As you will notice due to the rapid growth things in Babylon can and will change often and quickly. While this may look scary do not worry you are working with amazing people and everyone wants to help you. You have the right to ask questions and seek out help from people in the team and other co-workers. To make this task easier here’s, an incomplete, list of people that can help you.
+
+### Your manager
+Your manager will be the first person from the iOS team that you’ll see after the induction workshop on your first day. He, or she, should introduce you to your buddy, your squad and other co-workers.
+
+- [iOS Onboarding Line Manager Checklist](https://docs.google.com/spreadsheets/d/1Cu_e1Ol4nkpDNawQOeCbho9xg5L02-1k-Y1YCZjBZ_c/edit#gid=0)
+
+Why you should read it: This will tell you how your first 3 months at Babylon should look like.
```

Comment:
[nit] I'd add some formatting to `Why you should read it` everywhere

Reply:
great I will add it